hack/tests/verification-tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# Get verification tests source code prepare it and run the verification tests (Bushslicer)
+
+set -o pipefail
+
+. hack/tests/ci-prepare.sh
+
+function get_az_hosts() {
+  local IFS=$'\n'
+  for ip in $(az vmss list-instance-public-ips -g $RESOURCEGROUP -n ss-master --query "[].ipAddress" -o tsv) ; do
+    i+=${ip}':master:node,'
+  done
+  echo ${i%?}
+}
+
+BRANCH="${BRANCH:-v3}"
+VERIFICATION_TESTS_GIT="${VERIFICATION_TESTS_GIT:-https://github.com/openshift/verification-tests.git}"
+# Env vars for setting up test
+ARO_PUBLIC_HOSTNAME="${ARO_PUBLIC_HOSTNAME:-$(awk '/^  publicHostname:/{ print $2}' ${HOME}/go/src/github.com/openshift/openshift-azure/_data/containerservice.yaml)}"
```

Comment:
these calls that wrap awk in bash param expansion are perhaps too convoluted, can you move them to more clear `if` statements? e.g. `if [[ -z "$ARO_PUBLIC_HOSTNAME" ]]; then #do the awk magic here`

Reply:
I can refactor those if it will make it more readable. Other option is to make these come from the environment variables set by the caller of the script and simply exit if they are not provided. I'll also see if there is a less convoluted way to parse yaml in the shell.

Reply:
I'm not sure this will be more readable with a bunch of if statements, it's going to be a mess either way :)

I think that matching the exact spacing for the `publicHostname` here might be fragile though, can we match some arbitrary amount of whitespace there?